frontend/pages/admin/OrgSettingsPage/cards/constants.ts

```diff
@@ -12,7 +13,9 @@ export interface IAppConfigFormProps {
   appConfig: IConfig;
   isPremiumTier?: boolean;
   isUpdatingSettings?: boolean;
-  handleSubmit: (formUpdates: DeepPartial<IConfig>) => false | undefined;
+  handleSubmit: (formUpdates: DeepPartial<IConfig>) => Promise<boolean>;
+  router: InjectedRouter;
+  subsection?: string;
```

Comment:
We need the type on submit to allow us to set the dirty flag properly on update success/failure when that happens.

We need the router to navigate when tabs are changed.

We need subsection to pass through which tab to land on, e.g. for deep links from setup experience config.